src/models/piwikConfig.ts

```diff
@@ -2,4 +2,5 @@ export default interface PiwikConfig {
   url: string
   siteId: string
   landingPageStartGoalId: string
+  confirmationPagePDFGoalId: string
```

Comment:
shouldn't these be `number`s? CC @matthewdoreilly

Reply:
Yep, both `siteId` and `landingPageStartGoalId` should have been `number`.
Would you be able to update these types @faggc?

Reply:
Sure thing